src/views/guestLayout.js

```diff
@@ -42,6 +42,7 @@ const guestLayout = data => `
     <link rel='apple-touch-icon' href='/images/apple-touch-icon.webp' sizes='180x180'>
 
     <script src='/js/index.js' type='module'></script>
+    ${data.skipPartialModule ? '' : `<script src='/js/partials/${data.partial.name}.js' type='module'></script>`}
```

Comment:
What's the failure mode if someone forgets to set `skipPartialModule` but doesn't have client-side JS for that partial? Is it just a 404 in the console, or does anything break? If the former, then this looks fine to me, but if the latter, I wonder if we could replace it with a check that the file exists when the controller renders? Or maybe even better, checking which scripts are available when the server starts, to avoid disk access on every request.

But again, if it's just an extra 404, then that's no biggie :)

Reply:
it's just an extra 404 🤗

Reply:
I did initially have code in the module to fetch the url first and handle `!res.ok` before dynamically importing the module, but it seemed overkill.